src/parcsr_ls/par_amg_solveT.c

```diff
@@ -34,7 +34,7 @@ hypre_BoomerAMGSolveT( void               *amg_vdata,
    HYPRE_Int      amg_print_level;
    HYPRE_Int      amg_logging;
    HYPRE_Real  *num_coeffs;
-   HYPRE_Int     *num_variables;
+   HYPRE_BigInt     *num_variables;
```

Comment:
Should this be HYPRE_Real like in par_amg_solveT.c above? I don't think it makes a difference, but just asking for consistency.